allure-java-adaptor-api/src/main/java/ru/yandex/qatools/allure/utils/AnnotationManager.java

```diff
@@ -298,6 +306,16 @@ private String getTestCaseId() {
         return testCaseId == null ? null : testCaseId.value();
     }
 
+    private String getStoriesId() {
+        Stories stories = getAnnotation(Stories.class);
+
+        if(stories == null || stories.id().isEmpty()) {
```

Comment:
Mmm.. I guess you need 3 checks then: 
`if(stories == null || stories.id() == null || stories.id().isEmpty())`